src/app.d.ts

```diff
@@ -12,6 +12,8 @@ declare global {
 			openSigninModal?: VoidFunction;
 			email?: string;
 			password?: string;
+			isSignin?: boolean;
```

Comment:
@guacamole let's remove these variables. They're not being used

Reply:
got it!